conf.yaml

```diff
@@ -629,6 +629,10 @@ contents:
                 repo:   beats
                 path:   x-pack/filebeat/docs
                 exclude_branches:   [ 6.5, 6.4, 6.3, 6.2, 6.1, 6.0, 5.6, 5.5, 5.4, 5.3, 5.2, 5.1, 5.0, 1.3, 1.2, 1.1, 1.0.1 ]
+              -
+                repo:   beats
+                path:   x-pack/filebeat/module
```

Comment:
@karenzone @nik Can we just have one path here that's set to `x-pack/filebeat` instead of two?

Not sure if this will work or what the implications are on the doc build time.

Reply:
...if all the files at the specified path get cloned, then probably we *do* want to set both paths.